e2e/_suites/fleet/features/system_integration.feature

```diff
@@ -6,8 +6,8 @@ Scenarios for System Integration logs and metrics packages.
 Scenario Outline: Adding core System Integration to an Policy
   Given a "<os>" agent is deployed to Fleet with "tar" installer
     And the agent is listed in Fleet as "online"
-  When the policy is updated to have "system/metrics" set to "core"
-  Then "system/metrics" with "core" metrics are present in the datastreams
+  When the policy is updated to have "system/metrics" set to "core" for a "<os>" agent
```

Comment:
I believe you can obtain the "<os>" from the current installer. If not, we should provide that cc/ @adam-stokes

Reply:
I think the formatting stripped what was between the quotes, @mdelapenya are you talking about querying the underlying OS via installer?

Reply:
The installer has a reference to the downloadable file, so at that point we can set the target OS: "centos" or "debian", which was set by the example outline when deploying an "<os>" agent with "tar" installer

Reply:
Thinking about it, I think we could revert this https://github.com/elastic/e2e-testing/pull/1291/commits/16fa4ca4296fad72f73e4b31fdfbcc477d412d60 commit and the PR would be great to go.

Could you do it 🙏 ? We can discuss about anything else about it in a follow-up issue, so that we can see this code merged the soonest.

Reply:
This was not  the reason I put the os in there

Reply:
It is because when you run with new command it is not clear what os you running

Reply:
Yep, I saw the logs in the commit to print the "os". For that, and because the "os" is already set in the first step `a "<os>" agent is deployed to Fleet with "tar" installer`, I commented with @adam-stokes about storing that value internally (maybe in the installer struct), so that it's possible to retrieve it later without having to make the steps that expressive.

OTOH, we'd like to hear your intention about the change, so that we can better understand and review it

Reply:
Another thing that comes to my mind is that, if we keep the step as is now: `the policy is updated to have "system/metrics" set to "core" for a "<os>" agent` it could be the case that an scenario writer picking valid steps could create an scenario in a wrong manner:

_Omitting example outlines for simplicity:_
```gherkin
Given a "centos" agent is deployed to Fleet with "tar" installer 
Then the policy is updated to have "system/metrics" set to "core" for a "unexpected-value-not-centos" agent
```